torch_xla/distributed/xla_multiprocessing.py

```diff
@@ -202,7 +202,7 @@ def _prepare_env_for_index(index, pf_cfg):
   os.environ[xenv.LOCAL_ORDINAL] = str(index)
 
   if pf_cfg.dev_kind == 'TPU':
-    _setup_tpu_worker(index, gindex, pf_cfg, os.environ[xenv.TPU_CONFIG])
+    _setup_tpu_worker(index, gindex, pf_cfg, os.environ.get(xenv.TPU_CONFIG, None))
```

Comment:
I'd add an explicit `assert` at line 188, to check that `tpu_env_config` is not `None`.

Reply:
You mean, in addition to this change right?

Reply:
> You mean, in addition to this change right?

Yes, of course 😄

Reply:
Done, thanks!